index.html

```diff
@@ -0,0 +1,197 @@
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+            </ul>
+            <div class="search"></div>
+        </div>
+    </header>
+    <main class="main">
+        <div class="content">
+            <h1><span class="center-text">Some text here</span></h1>
+            <article class="story">
+                <div class="author">
+                    <p>James</p>
+                </div>
+                <div class="article-title">
+                    <p>Understanding CSS</p>
+                </div>
+                <div class="tags">
+                    <p>#something</p>
+                </div>
+            </article>
+
+            <article class="story light">
+                <div class="author">
+                    <p>James</p>
+                </div>
+                <div class="article-title">
+                    <p>Understanding CSS</p>
+                </div>
+                <div class="tags">
+                    <p>#something</p>
+                </div>
+            </article>
+
+            <article class="story dark">
+                <div class="author">
+                    <p>James</p>
+                </div>
+                <div class="article-title">
+                    <p>Understanding CSS</p>
+                </div>
+                <div class="tags">
+                    <p>#something</p>
+                </div>
+            </article>
+
+            <article class="story">
+                <div class="author">
+                    <p>James</p>
+                </div>
+                <div class="article-title">
+                    <p>Understanding CSS</p>
+                </div>
+                <div class="tags">
+                    <p>#something</p>
+                </div>
+            </article>
+        </div>
+        <div class="blue">
+            <div class="blue-button">
+
+            </div>
+        </div>
+        <div class="four-divs">
+            <section>
+            <h2>Title</h2>
+            </section>
+            <section>
+          <h2>Title</h2>
+            </section>
+            <section>
+            <h2>Title</h2>
+            </section>
+            <section>
+            <h2>Title</h2>
+            </section>
+        </div>
+        <div class="content red">
+            
+            <article class="red-square">
+                <div class="explore-features"></div>
+            </article>
+
+            <article class="twin-parent">
+                <div class="twincolumns"></div>
+                <div class="twincolumns"></div> 
+            </article>
+
+            <article class="twin-parent">
+                <div class="twincolumns"></div>
+                <div class="twincolumns"></div>
+            </article>
+
+            <article class="red-square">
+                <div class="explore-features"></div>
+            </article>
+        </div>
+
+        <div class="blue ad"></div>
```

Comment:
Let us improve semantics by using ```<article>``` instead of ```<div class="blue ad">```
![kr5](https://user-images.githubusercontent.com/46388113/74501410-f730cc80-4ee1-11ea-8f4e-8dec70e40007.PNG)